### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by edeandrea**

### CODE REVIEW

**Review by marchof — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ Twitter              | Fediverse                           | Name
 @clementplop         | @clementplop@mastodon.social        | Clement Escoffier
 @DaschnerS           | @daschners@mastodon.social          | Sebastian Daschner
 @delabassee          | @delabassee@mastodon.cloud          | David Delabassée
+@edeandrea.          | @edeandrea@mastodon.social.         | Eric Deandrea
```
[the remark this answers is not in the document]
Comment (edeandrea) [reply README.md:41]: Yeah that's weird no idea how that happened! I'll fix that in a bit. Sorry about that!

**Review by marchof — COMMENTED** [review]
Remove superfluous dots

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (marchof) [reply README.md]: Fixed it

**Review by marchof — APPROVED** [review]

**Comment by edeandrea** [comment]
Thank you!

**Pull request merged**